assets/js/services/helpers.js

```diff
@@ -42,6 +42,13 @@ export function getClassFromIngestSheetStatus(status) {
   return "";
 }
 
+export function getImageUrl(representativeImage) {
+  if (typeof representativeImage === "object") {
+    return representativeImage.url || "";
+  }
+  return representativeImage;
+}
+
```

Comment:
Can we add a test for this function?  What happens if `representativeImage` is undefined, or an empty string?   It'd be nice if this returned an empty string as a default.